.github/workflows/android-build.yml

```
@@ -0,0 +1,67 @@
+          distribution: 'zulu'
+          java-version: '8'
+
+      - name: Setup Android SDK
+        run: |
+          # Download and install a specific version of command line tools compatible with Java 8
+          mkdir -p ${ANDROID_HOME}/cmdline-tools
+          wget https://dl.google.com/android/repository/commandlinetools-linux-6609375_latest.zip
+          unzip -q commandlinetools-linux-6609375_latest.zip -d ${ANDROID_HOME}/cmdline-tools
+          mv ${ANDROID_HOME}/cmdline-tools/tools ${ANDROID_HOME}/cmdline-tools/latest
+          rm commandlinetools-linux-6609375_latest.zip
+          
```

YAMLlint (1.35.1)

[error] trailing spaces

(trailing-spaces)